frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts

```diff
@@ -0,0 +1,334 @@
+    },
+    VM_BOOTUP_TIMEOUT_SECS,
+  );
+
+  it(
+    'Migrate VM',
+    async () => {
+      await vm.action('Start');
+      const sourceNode = await vmDetailNode(vm.namespace, vm.name).getText();
+
+      await vm.action('Migrate');
+      await vm.waitForMigrationComplete(sourceNode, VM_MIGRATION_TIMEOUT_SECS);
+      expect(statusIcon(statusIcons.running).isPresent()).toBeTruthy();
+    },
+    VM_BOOT_AND_MIGRATE_TIMEOUT,
+  );
+
+  it(
+    'Migrate already migrated VM',
+    async () => {
+      await vm.action('Start');
+      let sourceNode = await vmDetailNode(vm.namespace, vm.name).getText();
+
+      await vm.action('Migrate');
+      await vm.waitForMigrationComplete(sourceNode, VM_MIGRATION_TIMEOUT_SECS);
+      sourceNode = await vmDetailNode(vm.namespace, vm.name).getText();
+
+      await vm.action('Migrate');
+      await vm.waitForMigrationComplete(sourceNode, VM_MIGRATION_TIMEOUT_SECS);
+      expect(statusIcon(statusIcons.running).isPresent()).toBeTruthy();
+    },
+    VM_BOOT_AND_MIGRATE_TIMEOUT * 2,
+  );
+
+  it(
+    'Cancel ongoing VM migration',
+    async () => {
+      await vm.action('Start');
+      const sourceNode = await vmDetailNode(vm.namespace, vm.name).getText();
+
+      // Start migration without waiting for it to finish
+      await vm.action('Migrate', false);
+      await vm.waitForStatusIcon(statusIcons.migrating, VM_MIGRATION_TIMEOUT_SECS);
+
+      await vm.action('Cancel', false);
+      await vm.waitForStatusIcon(statusIcons.running, VM_BOOTUP_TIMEOUT_SECS);
+      await browser.wait(
+        waitForStringInElement(vmDetailNode(vm.namespace, vm.name), sourceNode),
+        VM_MIGRATION_TIMEOUT_SECS,
+      );
+    },
+    VM_ACTIONS_TIMEOUT_SECS,
+  );
+});
+
+describe('Add/remove disks and NICs on respective VM pages', () => {
+  const testVm = getVmManifest('Container', testName, `vm-disk-nic-${testName}`);
+  const vm = new VirtualMachine(testVm.metadata);
+
+  beforeAll(async () => {
+    createResources([multusNad, testVm]);
+    await vm.action('Start');
+  }, VM_BOOTUP_TIMEOUT_SECS);
+
+  afterAll(() => {
+    deleteResources([multusNad, testVm]);
+  });
+
+  it(
+    'Add/remove disk on VM disks page',
+    async () => {
+      await vm.addDisk(hddDisk);
+      expect(await vm.getAttachedDisks()).toContain(hddDisk);
+
+      let vmi = await vm.navigateToVmi(TABS.OVERVIEW);
+      expect((await vmi.getVolumes()).includes(hddDisk.name)).toBe(false);
+
+      await vm.action('Restart');
+
+      vmi = await vm.navigateToVmi(TABS.OVERVIEW);
+      expect((await vmi.getVolumes()).includes(hddDisk.name)).toBe(true);
+
+      await vm.removeDisk(hddDisk.name);
+      expect(await vm.getAttachedDisks()).not.toContain(hddDisk);
+
+      await vm.action('Restart');
+
+      vmi = await vm.navigateToVmi(TABS.OVERVIEW);
+      expect((await vmi.getVolumes()).includes(hddDisk.name)).toBe(false);
+    },
+    VM_ACTIONS_TIMEOUT_SECS * 2,
+  );
+
+  xit(
```

Comment:
??

Reply:
`xit` skips a spec and highlights it in the test report as skipped.
It's skipped due to the opened BZ

Reply:
If possible, please link to the BZ blocking this test case via comment.